app/assets/javascripts/common/reports.js

```diff
@@ -1094,10 +1093,8 @@ function baseLineGraphConfig() {
       scales: {
         x: {
           stacked: false,
-          display: true,
```

Comment:
default display = true
removed as not required
https://www.chartjs.org/docs/latest/axes/#common-options-to-all-axes